spec/factories/sessions.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :session do
+    sequence :siteIP do |n|
+      fourth = n % 255
+      third = n / 255
+      "141.0.#{third}.#{fourth}"
```

Comment:
Are we returning an IP address?

Reply:
A sequence is every time you call it, you will get a different IP address.